### PR TITLE
Updates to deployment and new mainnet seed node URL

### PR DESCRIPTION
**Pull request by vepkenez**
**Type of PR:**
- [ ] Bugfix
- [ ] Feature
- [ ] Documentation
- [x] Other

**Required reviews:** 
- [x] 1
- [ ] 2
- [ ] 3

**What this does:**
* changes default mainnet seed node to new failover/loadbalanced routing at seeds.nucypher.network (documentation to follow this PR in knowledge base repo)
*  updates cloudworkers deploy/management to handle recent CLI and NuCypher code changes
* has been tested on live mainnet nodes
* adds a .dockerignore for more efficient and more secure docker deployments

**Why it's needed:**
> Nodes need to deploy and find each other with the greatest of ease.

### CODE REVIEW

**Comment by KPrasch** [comment]
Marking as draft until CI errors are resolved.

**Comment by vepkenez** [comment]
> Marking as draft until CI errors are resolved.

Those issues are happening on every PR right now.  I’m looking into it.

**Comment by vepkenez** [comment]
> > Marking as draft until CI errors are resolved.
> 
> Those issues are happening on every PR right now. I’m looking into it.

And they pass locally

File: newsfragments/2576.misc.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+new seed URL for mainnet seeds.nucypher.network
+cloudworkers updates
```
Comment (KPrasch) [inline newsfragments/2576.misc.rst:2]:
```suggestion
cloudworkers CLI updates
```

File: nucypher/utilities/templates/cloud_deploy_ansible_inventory.mako [hunk 2]
```diff
@@ -19,7 +19,7 @@ all:
                 node_is_decentralized: ${deployer.nodes_are_decentralized}
                 %if deployer.config.get('seed_node'):
                 SEED_NODE_URI: ${deployer.config['seed_node']}
-                teacher_options: --teacher ${deployer.config['seed_node']}
+                teacher_options: ""
```
Comment (KPrasch) [inline nucypher/utilities/templates/cloud_deploy_ansible_inventory.mako:22]: 👍🏻

**Review by KPrasch — APPROVED** [review]
🤠

**Pull request merged**